src/runtime/runtime_internal.h

```diff
@@ -207,9 +209,23 @@ typedef SinkPrinter debug;
 extern WEAK void halide_use_jit_module();
 extern WEAK void halide_release_jit_module();
 
+template <typename T>
```

Comment:
This file is only ever parsed by clang (it's not part of HalideRuntime.h), so it should be OK.